<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BinarizationDialog</class>
 <widget class="QDialog" name="BinarizationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Binarization</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="originalLabel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>450</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="originalLabelPart">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="binarizedLabel">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Wolf &amp; Jolion</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sauvola</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Otsu</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="local_widget" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>70</height>
           </size>
          </property>
          <widget class="QWidget" name="layoutWidget2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>331</width>
             <height>51</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_window">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Window size:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_kernel">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Strictness:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QSlider" name="windowSlider">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimum">
                 <number>2</number>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="singleStep">
                 <number>2</number>
                </property>
                <property name="value">
                 <number>40</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="kernelSlider">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>34</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1000</height>
           </size>
          </property>
          <widget class="QCheckBox" name="checkBox_advanced">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>201</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string>Advanced parameters:</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QWidget" name="advanced_widget" native="true">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>351</width>
             <height>91</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_9">
                <item>
                 <widget class="QCheckBox" name="checkBox_background">
                  <property name="text">
                   <string>Background 
subtraction:</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_enhancement">
                  <property name="text">
                   <string>Character 
enhancement:</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QSlider" name="backgroundSilder">
                  <property name="minimum">
                   <number>0</number>
                  </property>
                  <property name="maximum">
                   <number>60</number>
                  </property>
                  <property name="value">
                   <number>48</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="invertedAppearance">
                   <bool>false</bool>
                  </property>
                  <property name="invertedControls">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="enhancementSlider">
                  <property name="minimum">
                   <number>50</number>
                  </property>
                  <property name="maximum">
                   <number>150</number>
                  </property>
                  <property name="singleStep">
                   <number>10</number>
                  </property>
                  <property name="value">
                   <number>70</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="invertedAppearance">
                   <bool>false</bool>
                  </property>
                  <property name="invertedControls">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
          <property name="centerButtons">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BinarizationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BinarizationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
